packages/service/pyproject.toml

```diff
@@ -134,8 +134,6 @@ module = [
   "grpc.framework.foundation.*",
   # https://github.com/ni/hightime/issues/4 - Add type annotations
   "hightime.*",
-  # https://github.com/microsoft/tracelogging/issues/57 - Python traceloggingdynamic package is missing py.typed marker file
```

Comment:
Oops, this type-checks on Windows, but our GitHub Actions workflow runs mypy on Linux.

```
Run poetry run mypy
ni_measurement_plugin_sdk_service/_tracelogging.py:8: error: Cannot find implementation or library stub for module named "traceloggingdynamic"  [import-not-found]
ni_measurement_plugin_sdk_service/_tracelogging.py:8: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 2[14](https://github.com/ni/measurement-plugin-python/actions/runs/14848434595/job/41687298087?pr=1123#step:10:15) source files)
Error: Process completed with exit code 1.
```